src/sql/workbench/contrib/notebook/test/electron-browser/contentManagers.test.ts

```diff
@@ -131,12 +172,42 @@ suite('Local Content Manager', function (): void {
 		let displayOutput = <nb.IDisplayData>notebook.cells[0].outputs[0];
 		assert.equal(displayOutput.data['text/html'], '<div></div>');
 	});
+
 	test('Should create a new empty notebook if content is undefined', async function (): Promise<void> {
-		// verify that when loading content from an empty string or undefined, a new notebook is created.
+		// verify that when loading content from an empty string, a new notebook is created.
 		let content = await contentManager.loadFromContentString('');
+		assert.equal(content.metadata, undefined);
+		// verify that the notebook is empty
 		assert.equal(content.cells.length, 0);
+		assert.equal(content.nbformat, 4);
```

Comment:
Checking the format version here seems a little odd - ideally updating the version shouldn't break what we're checking for in the tests (or if it does that means the test needs to be updated)